tasks/get_record.json

```diff
@@ -12,6 +12,18 @@
         "sys_id": {
             "description": "ServiceNow sys_id",
             "type": "String[1]"
+        },
+        "user": {
+            "description": "ServiceNow username",
+            "type": "Optional[String[1]]"
+        },
+        "password": {
+            "description": "ServiceNow password",
+            "type": "Optional[String[1]]"
```

Comment:
Should this be SecureString?

Reply:
It doesn't look like SecureString will be an option.